src/components/editCenter/Description.js

```diff
@@ -52,6 +53,7 @@ const Description = ({step, setStep, result, setResult, clubs, setClubs}) => {
         updateCenter(result.id,result).then(response => {
             if(response.status && response.status ===  400 && response.data.message){
                 message.warning(response.data.message);
+                return;
             } else {
                 window.location.reload()
```

Comment:
Try to pass reload pagination page function in props. Do not use window.location.reload()